barretenberg/cpp/src/barretenberg/sumcheck/sumcheck.hpp

```diff
@@ -211,7 +211,7 @@ template <typename Flavor> class SumcheckProver {
             setup_zk_sumcheck_data(zk_sumcheck_data);
         };
 
-        bb::PowPolynomial<FF> pow_univariate(gate_challenges, multivariate_d);
+        bb::GateSeparatorPolynomial<FF> gate_separators(gate_challenges, multivariate_d);
```

Comment:
i would still preserve the univariate in the variable name for clarity but up to you

Reply:
I thought it was confusing because it's not actually an instance of  a Univariate class!